website/data/docs-nav-data.json

```diff
@@ -194,8 +194,8 @@
             "path": "concepts/client-count/counting"
           },
           {
-            "title": "Usage metrics",
-            "path": "concepts/client-count/usage-metrics"
+            "title": "Client usage",
+            "path": "concepts/client-count/client-usage"
```

Comment:
Thank you for updating this link! We also need to make sure to update all of the references to `monitoring/usage-metrics` - which I _think_ this maps to? You can double check by checking the docs site locally (see https://github.com/hashicorp/vault/blob/main/website/README.md#running-the-site-locally)